README.md

```diff
@@ -39,7 +39,11 @@ src="https://github.com/common-workflow-language/cwl-website/raw/main/openstand-
 The Common Workflow Language (CWL) is an open standard for describing analysis
 workflows and tools in a way that makes them portable and scalable across a
 variety of software and hardware environments, from workstations to cluster,
-cloud, and high performance computing (HPC) environments.  CWL is designed to
+cloud, and high performance computing (HPC) environments.
+
+CWL is for dataflow style batch analysis, where the units of processing are command line programs.
+
```

Comment:
Do we think it would be beneficial to comment here on some known workflows / use cases that CWL does NOT handle well? 

Example from chat:
```
"explicitly not for business process modeling"
```
And any other use cases that users can think of that aren't intended use cases?

Reply:
I would add something clarifying the use for batch processing vs interactive processing, as sometimes we've had confusion about workflows having being able to interact with external services such as as databases or other APIs

Reply:
> Do we think it would be beneficial to comment here on some known workflows / use cases that CWL does NOT handle well?

It could (and I see the value in that!), but it probably leaves the reader with a better feeling to not have a list of negatives when they first learn about something. I also don't want this introduction to be too long or wordy. A bit tricky to balance!

Reply:
Yeah, I can see where you are coming from there. It might also dissuade someone from trying it out if they do not fully understand what is meant by the item listed as "not supported" (IE cwl could be a use case for their problem, but since they don't understand the terminology of the item they might just not try it out). After thinking about it, it might do more harm than good.